server/models/Notifications.js

```diff
@@ -0,0 +1,26 @@
+const mongoose = require("mongoose");
+
+const notificationSchema = new mongoose.Schema({
```

Comment:
Question: How do we determine which notification belongs to which user? :)

Reply:
I added a recipient field inside the notifications model, which contains a string representing the user id of the recipient of the notification. I updated the getAllNotifications and getUnreadNotifications routes to reflect this change.